solr/core/src/test/org/apache/solr/security/AuditLoggerIntegrationTest.java

```diff
@@ -275,7 +275,7 @@ private void assertThreeAdminEvents() throws Exception {
 
   private static String AUTH_SECTION = ",\n" +
       "  \"authentication\":{\n" +
-      "    \"blockUnknown\":\"false\",\n" +
+      "    \"blockUnknown\":\"true\",\n" +
```

Comment:
Why does this explicit setting need to change? I cannot see that it changes anything elsewhere in the test?